### ISSUE TITLE
Workflow status icons aren’t visible in high-contrast mode with other themes than white 

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#9349]
⚠ This issue is specific to Windows High contrast mode / Contrast themes. To troubleshoot this, you will need to [simulate forced-colors mode](https://developer.chrome.com/blog/new-in-devtools-98/#forced-colors) in browser DevTools.

This is tagged as a [**good first issue**](https://github.com/wagtail/wagtail/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22) – if you want to take it on, please leave a comment below to say so, no need to ask for permission, first come first serve! State your intent and share any questions / plans you have on how to approach the task.

### Issue Summary

Discovered while testing #9305. Our workflow status icons in the workflow status dialog aren’t visible in forced colors mode, because of their white background not adapting to the theme. Here is what the modal looks like without forced colors:

![workflow-status-light](https://user-images.githubusercontent.com/877585/195651406-bb427c42-d635-4f0b-ae28-573698c6c873.png)

And then with a forced colors "dark" theme:

![workflow-status-dark](https://user-images.githubusercontent.com/877585/195651445-4b86a43f-7171-40ef-bcf1-8c0dbb8aed8d.png)

This should match the background of the page rather than stay white.

### Steps to Reproduce

1. Enable forced colors emulation in your browser
2. Edit a page and click "Submit for moderators approval"
3. Go back to the Edit UI and in the "Info" side panel, click "View details"

### Proposed solution

It looks like this could be a simple change, but this needs further testing.

Here is where we set the background color: https://github.com/wagtail/wagtail/blob/main/client/scss/components/_workflow-timeline.scss#L43

After those few declarations, we should be able to add:

```css
@media (forced-colors: active) {
      background: Canvas;
}
```

**Comment by Lovelyfin00** [comment]
I would love to work on this too

**Comment by thibaudcolas** [comment]
@Lovelyfin00 only one issue at a time please!

**Comment by Lovelyfin00** [comment]
Fixed in wagtail/wagtail#9349

I didn;t know I had to create separate branches for each PR I make
